recipes/libta-lib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libta-lib" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-src.tar.gz  # [unix]
+    sha256: 9ff41efcb1c011a4b4b6dfc91610b06e39b1d7973ed5d4dee55029a0ac4dc651                    # [unix]
+    folder: unix                                                                                # [unix]
+  - url: https://downloads.sourceforge.net/project/ta-lib/ta-lib/0.4.0/ta-lib-0.4.0-msvc.zip
+    sha256: 2dae7a1391222f0a7e601ecda654f93cebedfdb37afeb30df2721a3b0a717875
+    folder: win
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libta_lib${SHLIB_EXT}     # [unix]
+    - test -f $PREFIX/include/ta-lib/ta_abstract.h  # [unix]
+    - test -f $PREFIX/include/ta-lib/ta_common.h    # [unix]
+    - test -f $PREFIX/include/ta-lib/ta_defs.h      # [unix]
+    - test -f $PREFIX/include/ta-lib/ta_func.h      # [unix]
+    - if exist %LIBRARY_INC%\ta_abstract.h (exit 0) else (exit 1)         # [win]
+    - if exist %LIBRARY_INC%\ta_common.h (exit 0) else (exit 1)           # [win]
+    - if exist %LIBRARY_INC%\ta_defs.h (exit 0) else (exit 1)             # [win]
+    - if exist %LIBRARY_INC%\ta_func.h (exit 0) else (exit 1)             # [win]
+    - if exist %LIBRARY_LIB%\ta_abstract_cdd.lib (exit 0) else (exit 1)   # [win]
+    - if exist %LIBRARY_LIB%\ta_common_cdr.lib (exit 0) else (exit 1)     # [win]
+    - if exist %LIBRARY_LIB%\ta_func_cmd.lib (exit 0) else (exit 1)       # [win]
+    - if exist %LIBRARY_LIB%\ta_libc_csd.lib (exit 0) else (exit 1)       # [win]
+    - if exist %LIBRARY_LIB%\ta_libc_csr.lib (exit 0) else (exit 1)       # [win]
+
+about:
+  home: https://ta-lib.org/index.html
+  summary: "TA-Lib : Technical Analysis Library"
+  license: BSD-2-Clause
+  # License is included in windows source download, but not unix:
+  license_file: win/LICENSE.TXT
+
+extra:
+  recipe-maintainers:
+    - gerrymanoim
+    - llllllllll
```

Comment:
@gerrymanoim @llllllllll - are you OK with being maintainers?

Reply:
Yep - sorry missed this. All good.

Reply:
I am.